lib/person.rb

```diff
@@ -0,0 +1,30 @@
+require 'yaml'
+require 'date'
+require './lib/library.rb'
+
+def books
+  YAML.load_file('./lib/books.yml')
```

Comment:
You should access the collection of books through an instance of the Library class, not read them directly from the data file (yaml)